.github/workflows/rust-ignored.yml

```diff
@@ -113,3 +113,10 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - run: 'echo "Rust - lint_cargo_lock - Ignored"'
+
+  test_orchestrator_ockam_command:
+    name: Rust - test_orchestrator_ockam_command
+    runs-on: ubuntu-20.04
+    container: ghcr.io/build-trust/artifacts-helper:latest
```

Comment:
Can we please omit the `container` if the only steps are a no-op `echo`, so that it doesn't spend time and bandwith to pull a custom image until one is functionally needed?